src/index.js

```diff
@@ -1,12 +1,12 @@
 module.exports = ({ types: t }) => {
-  const minifyRaw = (raw) => {
+  const minifyValue = (raw) => {
```

Comment:
Could you rename the `raw` argument e.g. `value`? After that I think this is good to go.